openwisp_monitoring/device/static/monitoring/css/daterangepicker.css

```
@@ -74,7 +74,7 @@
   top: initial;
   bottom: -6px;
   border-bottom: initial;
-  border-top: 6px solid #fff;
+  border-top: 6px solidvar(--ow-color-white);
```

Biome (2.4.7)

[error] Unexpected unknown function: solidvar

(lint/correctness/noUnknownFunction)